challenges/objects-arrays.js

```diff
@@ -89,37 +119,51 @@ The zoo wants to display both the scientific name and the animal name in front o
 
 */
 const animalNames = [];
+
+let fullNames = zooAnimals.forEach(function(i){
+  animalNames.push(i.animal_name + " " + i.scientific_name);
+});
```

Comment:
you could use a template string here to and not have to add in the extra space between the animal name and the scientific name